selfdrive/test/test_routes.py

```diff
@@ -85,6 +85,7 @@
   TestRoute("50a2212c41f65c7b|2021-05-24--16-22-06", HYUNDAI.KIA_FORTE),
   TestRoute("c5ac319aa9583f83|2021-06-01--18-18-31", HYUNDAI.ELANTRA),
   TestRoute("82e9cdd3f43bf83e|2021-05-15--02-42-51", HYUNDAI.ELANTRA_2021),
+  TestRoute("715ac05b594e9c59|2021-06-20--16-21-07--8", HYUNDAI.ELANTRA_HEV_2021),
```

Comment:
this route isn't fully uploaded. make sure you have the "Upload Raw Logs" toggle enabled in the settings and connect your device to wifi until it uploads

Reply:
also remove the segment, like this:
```suggestion
  TestRoute("715ac05b594e9c59|2021-06-20--16-21-07", HYUNDAI.ELANTRA_HEV_2021),
```

Reply:
Gotcha. The "Upload Raw Logs" toggle was on when I test drive it, so I think that's okay. I just went to `useradmin` to check, and I believe it is now. It may have not been when I initially took the PR off draft.

There is a hyperlink in the interface that says "make public." Do you think this is necessary for CI to succeed? 
<img width="549" alt="Screen Shot 2021-06-21 at 7 12 16 AM" src="https://user-images.githubusercontent.com/21319730/122776765-3f8f9580-d260-11eb-9bad-0e9a6c3204e2.png">